### PR TITLE
[8008]Workflow404cssfiles

### LINKED ISSUE #8008: 404 error in workflow edit page for WorkflowsActivity-*.css files
1. go to workflow list page
2. edit or create one
3. F12
4. 404 error on WorkflowsActivity-*.css files

### PR DESCRIPTION
**Pull request by HengzheLi**
fixes #8008

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs [hunk 1]
```diff
@@ -18,13 +18,18 @@ public class ResourceManifest : IResourceManifestProvider {
 
             manifest.DefineStyle("WorkflowsAdmin").SetUrl("orchard-workflows-admin.css").SetDependencies("~/Themes/TheAdmin/Styles/Site.css");
 
-            var activityNames = _activitiesManager.Value.GetActivities().Select(x => "WorkflowsActivity-" + x.Name).ToArray();
-
-            foreach (var activityName in activityNames) {
-                manifest.DefineStyle(activityName).SetUrl(activityName.HtmlClassify()).SetDependencies("WorkflowsAdmin");
+            var activities = _activitiesManager.Value.GetActivities().ToArray();//.Select(x => "WorkflowsActivity-" + x.Name).ToArray();
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs]: Remove the comment

File: src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs [hunk 2]
```diff
@@ -18,13 +18,18 @@ public class ResourceManifest : IResourceManifestProvider {
...
+
+            foreach (var activity in activities) {
+                var assemblyName = activity.GetType().Assembly.GetName().Name;
+                var styleName = "WorkflowsActivity-" + activity.Name;
+                manifest.DefineStyle(styleName)
+                    .SetBasePath(VirtualPathUtility.AppendTrailingSlash("~/Modules/"+assemblyName+"/styles/"))
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs]: Indentation and spaces around `+`

File: src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs [hunk 3]
```diff
@@ -18,13 +18,18 @@ public class ResourceManifest : IResourceManifestProvider {
...
+                    .SetUrl(styleName.HtmlClassify()+".css")
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs]: Spaces

**Comment by barthamark** [comment]
Please decline this PR since the source repository doesn't exist anymore. The changes are re-implemented in the Lombiq fork and a PR was created. #8201

**Comment by BenedekFarkas** [comment]
@HengzheLi @sebastienros I'll close this as #8201 solves the issues this PR was submitted for, but also improves the original implementation.

**Pull request closed without merging**